DigitalOffice/Services/ProjectService/Mappers/ProjectMapper.cs

```diff
@@ -35,6 +35,24 @@ public DbProject Map(NewProjectRequest request)
                 DepartmentId = request.DepartmentId,
                 Description = request.Description,
                 IsActive = request.IsActive,
+                Name = request.Name,
+                Deferred = false
+            };
+        }
+
+        public DbProject Map(EditProjectRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            return new DbProject
+            {
+                DepartmentId = (Guid)request.DepartmentId,
+                Name = request.Name,
+                Description = request.Description,
+                IsActive = (bool)request.IsActive,
```

Comment:
What happens if there is null?

Reply:
Well, we have to add ? and in the DbEntities? Can do, but should I?
What do you think

Reply:
Needs to be discussed at a meeting